specifyweb/frontend/js_src/lib/components/Forms/Save.tsx

```diff
@@ -195,7 +201,10 @@ export function SaveButton<SCHEMA extends AnySchema = AnySchema>({
   const copyButton = (
     label: LocalizedString,
     description: LocalizedString,
-    handleClick: () => Promise<SpecifyResource<SCHEMA>>
+    handleClick: () =>
+      | Promise<readonly SpecifyResource<SCHEMA>[]>
+      | Promise<SpecifyResource<SCHEMA>>,
```

Comment:
Does `Promise<SpecifyResource<SCHEMA>>` need to be readonly as well? Also, would wrapping `<readonly SpecifyResource<SCHEMA>[]` in `RA<>` be more consistent?